assets/dev/js/editor/document/elements/commands/styles.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * @typedef {import('../../../container/container')} Container
```

Comment:
Do we need it?

Reply:
Yes it's for the JSDoc comment for `addToHistory` (line 37)

```
	 /**
	 * Function addToHistory().
	 *
	 * @param {Container} container
	 * @param {{}}        newStyles
	 * @param {{}}        oldStyles
	 */
```